GliaWidgets/Lib/Presenter/GliaPresenter.swift

```diff
@@ -0,0 +1,34 @@
+import UIKit
```

Comment:
Let's add a new line between import and class definition.

Reply:
Whoops. Fixed